network-api/networkapi/wagtailpages/templates/buyersguide/product_page.html

```diff
@@ -235,9 +235,9 @@ <h3 class="tw-mb-4 tw-font-zilla tw-text-2xl tw-leading-7">{% trans "What data d
                 </div>
 
                 <div class="tw-w-full tw-flex-shrink-0" data-product-label="1">
-                  {% localizedroutablepageurl home_page 'min-security-view' as min_sec_url %}
+                  {% localizedroutablepageurl home_page 'methodology-view' as min_sec_url %}
                   {% blocktrans with url=min_sec_url asvar minimum_security_standards trimmed %}
```

Comment:
Let’s concatenate the anchor with the url var, so that we don’t need to invalidate the translation now and in the future if this ever change again.

```suggestion
                  {% blocktrans with url=min_sec_url|add:"#minimum-security-standards" asvar minimum_security_standards trimmed %}
```